tools/release-ci/pkg/config/save.go

```diff
@@ -105,6 +113,19 @@ func retrieveRepoConfig() (Repo, bool, error) {
 		missing = append(missing, "REPO_NAME")
 	}
 
+	if len(missing) > 0 {
```

Comment:
These are not going to be set for periodics, so this is not always an error

Reply:
Our pipeline will always set it